dask/dataframe/core.py

```diff
@@ -4195,7 +4195,9 @@ def select_dtypes(self, include=None, exclude=None):
         cs = self._meta.select_dtypes(include=include, exclude=exclude).columns
         return self[list(cs)]
 
-    def sort_values(self, by, npartitions=None, ascending=True, **kwargs):
+    def sort_values(
+        self, by, npartitions=None, ascending=True, na_position="last", **kwargs
```

Comment:
Can you add `na_position` to the parameters section of the docstring below?

Reply:
Done! Also changed the description for `ascending` to reflect #8130